src/test/java/seedu/address/model/InternshipModelManagerTest.java

```diff
@@ -132,4 +136,24 @@ public void equals() {
         differentUserPrefs.setInternshipFilePath(Paths.get("differentFilePath"));
         assertFalse(internshipModelManager.equals(new InternshipModelManager(internshipBook, differentUserPrefs)));
     }
+
+    @Test
+    public void createInternship_internshipsAreSortedByComparator() {
+        internshipModelManager.createInternship(JANESTREET);
+        internshipModelManager.createInternship(CITADEL);
+        ObservableList<Internship> internships = internshipModelManager.getFilteredInternshipList();
+        // Assuming the default comparator is by company name
+        assertEquals(internships.get(0), CITADEL);
+        assertEquals(internships.get(1), JANESTREET);
+    }
```

Comment:
Love this test as it ensures that our default sorting order is correct - by company name